### PR TITLE
Documentation linting, Group Examples and Windows support

### PR DESCRIPTION
**Pull request by Naishy**
Sorry for mixing this all in one PR! All minor changes though.

- Documentation linting
The readme.md was pulling your code rating down so I ran it through a linter when I did the examples for Group.
- Group Examples
Adding some basic example for how to use the Group functionality
- Windows support
The URL building wasn't working for me on Windows - it was using the wrong sort of slashes. Adding the trailing slash (which is what the browser does anyway) fixed it but it also seemed to make sense to use the urljoin rather than the OS pjoin function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,37 +143,90 @@ Please, be aware that is not documented in the official API.
 
 ### Groups
 
-Need documentation...
+#### Get all Groups
+
+```python
+>>> api.groups.all()
+>>> api.groups.all(limit=50)
+>>> api.groups.all(ofset=10)
```
Comment (skoudoro) [inline README.md:151]: typo: `offset`

File: mailerlite/client.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def make_request(url, method, headers=None, data=None,
         raise ValueError("Incorrect request method. method should be "
                          "{}".format(VALID_REQUEST_METHODS))
 
-    url = pjoin(MAILERLITE_API_V2_URL, url)
+    url = urljoin(MAILERLITE_API_V2_URL, url)
```
Comment (skoudoro) [inline mailerlite/client.py:93]: Good Catch! I need to add Windows CI's via GitHub action to catch this kind of error.

File: README.md (hunk 1 above)
Comment (Naishy) [reply README.md]: Doh! Good spot!

File: mailerlite/client.py (hunk 2 above)
Comment (Naishy) [reply mailerlite/client.py:92]: Also just removed the import that wasn't needed any more

**Pull request merged**